internal/kojiapi/server.go

```diff
@@ -60,6 +60,17 @@ type apiHandlers struct {
 
 // PostCompose handles a new /compose POST request
 func (h *apiHandlers) PostCompose(ctx echo.Context) error {
+	tls := ctx.Request().TLS
+	if tls == nil {
+		panic("kojiapi must be always run on TLS listener")
+
+	}
+	if tls.PeerCertificates == nil || len(tls.PeerCertificates) > 0 {
+		panic("kojiapi must be always run with required peer certificates")
+	}
+	// take the job owner name from the first SAN name
+	jobOwner := (*tls.PeerCertificates[0]).DNSNames[0]
```

Comment:
Are we guaranteed that this is set? We no longer support CN? (I would not mind deprecating CN, but should make sure our our issued certs are up-to-date).